components/brave_wallet_ui/common/hooks/send.ts

```diff
@@ -367,7 +367,7 @@ export default function useSend () {
     } else if (selectedAccount?.coin === BraveWallet.CoinType.SOL) {
       processSolanaAddress(toAddressOrUrl)
     }
-  }, [toAddressOrUrl, selectedAccount?.coin, ensOffchainLookupOptions])
+  }, [toAddressOrUrl, selectedAccount, ensOffchainLookupOptions])
```

Comment:
The real issue is that we didn't include `processEthereumAddress`, `processFilecoinAddress` and `processSolanaAddress` in this dependency array, as these functions are redefined when the selected account changes.

Reply:
Updated!